### ISSUE TITLE
Fix OpenGL Core for OSX

### ISSUE THREAD

**Issue by billyquith** [billyquith/GWork#79]
Fails, as described here: https://github.com/glfw/glfw/issues/876

- Operating System (name & version): OSX 10.12
- Compiler (name & version): clang
- Target renderer: OpenGL Core
- cmake command-line used: `-GXcode -DRENDER_OPENGL_CORE=ON`

**Comment by billyquith** [comment]
This looks to be a problem with GLFW3, perhaps fixed in (unreleased) 3.3.

@Megaxela Does this work ok on other platforms❓

**Comment by Megaxela** [comment]
Yes it's working on Windows (checked with AMD R9 380) and Linux (Intel
integrated)
I guess this issue must stay open until you'll be able to check it with new
GLFW (it's almost released). Or find any fix (without patching glfw ofc)
